### PR TITLE
added info about discussions and link to sandboat wiki

### PR DESCRIPTION
**Pull request by kaincenteno**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -45,6 +45,21 @@
         <li><a href="https://github.com/kaincenteno/wiki">wiki Github</a>
       </ul>
 
+    <h4>FOUND A BUG IN THE GAME</h4>
+      <p>If you have found a bug in the game. We encourage you to submit it through a discussion.
```
Comment (EpicTaru) [inline index.php:49]: Formatting, suggest this is changed to:
We encourage you to submit it through a discussion. If you have found a bug in the game:

File: index.php [hunk 2]
```diff
@@ -45,6 +45,21 @@
...
+      </p>
+      <ul>
+        <li><a href="https://github.com/LandSandBoat/server/discussionsi">LandSandBoat Server Github Discussion</a></li>
```
Comment (EpicTaru) [inline index.php:52]: Change link address to: https://github.com/LandSandBoat/server/discussions

File: index.php [hunk 3]
```diff
@@ -45,6 +45,21 @@
...
+      </ul>
+
+    <h4>RUN YOUR OWN SERVER</h4>
+      <p>You can find all the information needed to run your own private server by going to the projects
```
Comment (EpicTaru) [inline index.php:56]: Punctuation, suggest this is changed to:
You can find all the information needed to run your own private server by going to the project's

File: index.php [hunk 4]
```diff
@@ -45,6 +45,21 @@
...
+        github wiki. We (xiweb) do not provide support for setting up your own server.
```
Comment (EpicTaru) [inline index.php:57]: Are we xiweb or XIWeb?

File: index.php [hunk 5]
```diff
@@ -45,6 +45,21 @@
...
+      </p>
+      <ul>
+        <li><a href="https://github.com/LandSandBoat/server/wiki">LandSandBoat Server Github Wiki</a></li>
+      </ul>
+
     <h4>CONTRIBUTE UPSTREAM(LANDSANDBOAT REPO)</h4>
```
Comment (EpicTaru) [inline index.php:63]: Punctuation, missing a space between UPSTREAM and (LANDSANDBOAT, change to:
<h4>CONTRIBUTE UPSTREAM (LANDSANDBOAT REPO)</h4>

File: index.php [hunk 6]
```diff
@@ -45,6 +45,21 @@
...
       <p>Upstream has high standards on the code being merged. PLEASE PLEASE PLEASE, check past PRs and read their
```
Comment (EpicTaru) [inline index.php:64]: Punctuation, comma not necessary between PLEASE and check, change to:
PLEASE PLEASE PLEASE check

File: index.php [hunk 7]
```diff
@@ -45,6 +45,21 @@
...
         guidelines before submitting any work. We do not want anyone's feelings hurt, and/or their time wasted.
```
Comment (EpicTaru) [inline index.php:65]: Punctuation, comma not necessary between hurt and and/or, change to:
hurt and/or their time wasted.

**Review by EpicTaru — CHANGES_REQUESTED** [review]

File: index.php (hunk 4 above)
Comment (kaincenteno) [reply index.php:57]: xiweb all lowercase. Changed it on the "WELCOME TO part" to match hehe thanks for checking

File: index.php (hunk 6 above)
Comment (kaincenteno) [reply index.php:64]: nice catch, thought you were gonna tell me to add more PLEASE :trollface:  lol

**Pull request merged**